tools/cpplint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
we have lots of other python files...

Reply:
Sure, I'll go fix those too. I only noticed this one, since it was being used in a git hook in a different repo.